runtime.go

```diff
@@ -27,6 +33,11 @@ type Runtime interface {
 	LookupBuiltinType(pkg, name string) (semantic.MonoType, error)
 }
 
+type ASTHandle interface {
```

Comment:
Can you document this interface, I think I understand its usage, but its not completely clear?

Reply:
Yes, I added a comment.  It's needed to avoid a dependency from `flux` on `libflux/go/libflux`.